src/main/kotlin/com/cognifide/gradle/aem/instance/tail/InstanceTailer.kt

```diff
@@ -73,6 +74,21 @@ class InstanceTailer(val aem: AemExtension) {
                     ?.let { aem.project.file(it) }
                     ?: File(aem.configCommonDir, "instanceTail/incidentFilter.txt")
 
+    /**
+     * Indicates if tailer will print logs of specified instances to console.
+     *
+     * When no instances are specified, it searches for the first author
+     */
+    var instancesToFollow = when (aem.project.hasProperty("instance.tail.follow")) {
```

Comment:
InstancesFollowed / I am not using prepositions anywhere else

Reply:
Avoid nulls. Introduce separate boolean instanceFollowing: boolean
InstancesFollowed : list<instance>

And you avoided null at the beginning.. And you don't need to check it later. Try to design code in that way / eliminating null types as much as it is possible